c_api/include/taichi/taichi.hpp

```diff
@@ -0,0 +1,652 @@
+// C++ wrapper of Taichi C-API
```

Comment:
Are we gonna release a C++ wrapper together with C-API?

Reply:
Y